support-frontend/assets/components/headers/header/header.scss

```diff
@@ -318,3 +318,11 @@ $header-links-min-width: gu-span(12);
     display: none;
   }
 }
+
+.component-header-links__li.show-on-tablet {
```

Comment:
maybe this should be a modifier looking for an additional class

`.component-header-links__li--show-on-tablet`